# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/berkeley-abc/install.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff --git a/Makefile b/Makefile
index 798fb2e7b..e595e1055 100644
--- a/Makefile
+++ b/Makefile
@@ -243,3 +243,10 @@ cmake_info:
 	@echo SEPARATOR_CXXFLAGS $(CXXFLAGS) SEPARATOR_CXXFLAGS
 	@echo SEPARATOR_LIBS $(LIBS) SEPARATOR_LIBS
 	@echo SEPARATOR_SRC $(SRC) SEPARATOR_SRC
+
+install: $(PROG) lib$(PROG).a lib$(PROG).so
+	rsync -avm --include='*.h' --include='*/' --exclude='*' src/ /usr/include/abc/
+	cp $(PROG) /usr/bin/$(PROG)
+	chmod +x /usr/bin/$(PROG)
+	cp lib$(PROG).a /usr/lib64/lib$(PROG).a
+	cp lib$(PROG).so /usr/lib64/lib$(PROG).so
